indico/modules/categories/blueprint.py

```diff
@@ -32,6 +32,7 @@
 _bp.add_url_rule('/manage/protection', 'manage_protection', RHManageCategoryProtection, methods=('GET', 'POST'))
 _bp.add_url_rule('/manage/settings', 'manage_settings', RHManageCategorySettings, methods=('POST', 'GET'))
 _bp.add_url_rule('/manage/sort-subcategories', 'sort_subcategories', RHSortSubcategories, methods=('POST',))
+_bp.add_url_rule('/manage/events/<int:page_no>', 'paginated_events_list', RHCategoryEventsListPaginated)
```

Comment:
`/manage/events` and then `?page=...&order=...` in the query string (for now with ajax, if we decide to do it without ajax this logic would go into the `/manage/` RH)